### PR TITLE
Adding mentors when create StudentGroups

### PR DESCRIPTION
**Pull request by Vlad1023**
Fixed issue: ability to add mentors when create new StudentGroup

### CODE REVIEW

**Review by SerjKocherga — APPROVED** [review]

**Review by SnipeFrost — COMMENTED** [review]
better using IList or IEnumerable

**Comment by sober607** [comment]
All ok, but try to use IEnumerable or IList returning data type

**Review by sober607 — APPROVED** [review]
All ok, but try to use IEnumerable or IList returning data type

**Comment by Vlad1023** [comment]
> All ok, but try to use IEnumerable or IList returning data type

As it said here https://stackoverflow.com/questions/26676563/entity-framework-queryable-async better to use List return type with async

File: CharlieBackend.Business/Services/StudentGroupService.cs [hunk 1]
```diff
@@ -47,6 +47,20 @@ public async Task<StudentGroupModel> CreateStudentGroupAsync(CreateStudentGroupM
                     }
                 }
 
+                if(studentGroupModel.MentorIds.Count != 0)
```
Comment (whereismyroot) [inline CharlieBackend.Business/Services/StudentGroupService.cs:50]: MentorIds could be null here

**Review by whereismyroot — CHANGES_REQUESTED** [review]
Add null check

File: CharlieBackend.Business/Services/StudentGroupService.cs (hunk 1 above)
Comment (Vlad1023) [reply CharlieBackend.Business/Services/StudentGroupService.cs]: done

File: CharlieBackend.Business/Services/StudentGroupService.cs [hunk 2]
```diff
@@ -47,6 +47,20 @@ public async Task<StudentGroupModel> CreateStudentGroupAsync(CreateStudentGroupM
                     }
                 }
 
+                if(studentGroupModel?.MentorIds.Count != 0)
```
Comment (whereismyroot) [inline CharlieBackend.Business/Services/StudentGroupService.cs:50]: besides studentGroupModel, MentorIds could also be null. It is better to check studentGroupModel at the beggining of the method, and throw ArgumentNullException.

**Review by whereismyroot — APPROVED** [review]

File: CharlieBackend.Business/Services/StudentGroupService.cs (hunk 2 above)
Comment (Vlad1023) [reply CharlieBackend.Business/Services/StudentGroupService.cs:50]: But in type of studentGroupModel (CreateStudentGroupModel) we have data attribute field [Required] for MentorIds

**Pull request merged**